### PR TITLE
Show Send Tip form at any time

### PR DESCRIPTION
**Pull request by stanislav-slavov**
ref #398

### CODE REVIEW

**Comment by stanislav-slavov** [comment]
I have added an orange warning message. 
The whole form is disabled and the warning is shown if there is no wallet or the user is not logged.
I don't think we should display the warning if everything is ok with the account.

File: src/styles/_variables.scss [hunk 1]
```diff
@@ -15,11 +15,10 @@ $thumbnail_background_color_alt: #33343e;
...
 
-//Custom styles
 $custom_links_color: #67f7b8;
 $search_icon_hover_color: $white_color;
 $search_icon_backgound: #52528b;
-$article_description_color: #96969b;
 $comment_text_color: #8f9095;
 $tip_note_color: #babac0;
 $text_content_color: #d2d2d4;
+$warning_font_color: #856404;
```
Comment (venimus) [inline src/styles/_variables.scss:24]:
```suggestion
$warning_font_color: #ffdc50;
```
We need to use only design consistent colors. This one is approved for warning tough it is not yet implemented elsewhere. It is for "site badges" - warning, we can use it sitewise

Comment (stanislav-slavov) [reply src/styles/_variables.scss:24]: Thanks for providing it.

File: src/components/layout/SendTip.vue [hunk 2]
```diff
@@ -27,15 +27,25 @@
...
+            v-model="sendTipForm.amount"
+            :disabled="!canTip"
+          />
         </div>
       </div>
+      <div
+        v-if="!canTip"
+        class="install-wallet-warning"
+      >
+        The wallet should be installed and active in order to tip.
```
Comment (mradkov) [inline src/components/layout/SendTip.vue:44]:
```suggestion
        You need to have a wallet installed and active in order to tip.
```

**Pull request merged**